src/handlers/imgix.ts

```diff
@@ -36,10 +36,7 @@ const staticImgixClient = shouldCreateImgixClient();
 // for components which may have been unmounted/remounted. This is done to
 // increase performance.
 
-// TODO: We need to find a suitable upper limit.
-//       This might be conditional based upon either the runtime
-//       hardware or the number of unique tokens a user may have.
-const capacity = 256;
+const capacity = 512;
```

Comment:
I thnk this comment is still valid, This is not more suitable than 256, just 2x bigger

Reply:
Maybe even 1000 would be okay, we should do this looking at hardware or something. Anyway, please restore the comment. OK to bump!

Reply:
got it will do